src/hammer-vlsi/technology/asap7/defaults.yml

```diff
@@ -58,6 +58,10 @@ technology.core:
   # This is used to provide a reference master for generating standard cells
   tap_cell_rail_reference: "{TAPCELL*}"
 
+synthesis.inputs:
+  latch_map_file: "${vlsi.builtins.hammer_vlsi_path}/technology/asap7/extra/latch_map.v"
```

Comment:
This key needs to be added to `hammer-vlsi/defaults.yml` and `hammer-vlsi/defaults_types.yml`

Reply:
Actually, on another thought, I think this should be a `synthesis.yosys` key since the format isn't understood by any other tool.

Reply:
I see you changed it elsewhere but it needs to be `synthesis.yosys` in this file too